plugins/quota/actor.go

```diff
@@ -0,0 +1,105 @@
+package quota
+
+import (
+	"os"
+	"path"
+
+	"github.com/gluster/glusterd2/glusterd2/daemon"
+	"github.com/gluster/glusterd2/glusterd2/volume"
+	"github.com/gluster/glusterd2/glusterd2/xlator"
+	gderrors "github.com/gluster/glusterd2/pkg/errors"
+	log "github.com/sirupsen/logrus"
+)
+
+type quotadActor struct{}
+
+const (
+	quotaDaemonKey = "enable"
+)
+
+// IsQuotadAffectedNode returns true if there are local bricks of volume on
+// which quota is enabled
+func IsQuotadAffectedNode() bool {
```

Comment:
Do not export this function is it's not used outside of this package.

Reply:
done